### ISSUE TITLE
Reader: Alt text on the featured image is not showing in the Feed on Reader

### ISSUE THREAD

**Issue by galakhyati** [Automattic/wp-calypso#49226]
### Steps to reproduce the behavior

1. From WordPress.com > My Sites > Site > Media add alt text on an image.
2. Assign this image (with alt text) as a post's Featured Image.
3. Check the post in the Reader, the alt text added to the featured image is not showing when checked in the browser console.

#### What I expected to happen
I was expecting the alt text to be showing on the image in the Reader as on individual posts.

#### What actually happened
The alt text is not showing on the featured image on Reader.

### Context
A user reported how to add alt text to the image that is showing on the Reader. 

#### Does this happen on simple or atomic sites or both?
Both

#### Is there any console output or error text?
Nope

#### Screenshot / Video: If applicable, add screenshots to help explain your problem.
Screencast of a simple site featured image alt text shows in Reader and on an individual post: https://d.pr/v/SzlnsQ/Kbr3SACApB

#### Context / Source

Ticket: #3661445-zd

**Comment by ivan-ottinger** [comment]
I was able to reproduce the issue. The ALT text is missing in the Reader:

![Markup on 2021-08-09 at 10:02:25](https://user-images.githubusercontent.com/25105483/128675853-59810cb7-1935-4b56-9be5-abbda917b313.png)

I am going to investigate the issue further.

**Comment by ivan-ottinger** [comment]
Issue is fixed by https://github.com/Automattic/wp-calypso/pull/55378

Please feel free to review the above PR to see the implementation details.
